src/gwt/src/org/rstudio/studio/client/projects/StudioClientProjectConstants.java

```diff
@@ -0,0 +1,2142 @@
+package org.rstudio.studio.client.projects;
```

Comment:
This needs our standard file header block

Reply:
Is that not the correct file header block?

Reply:
Oops, sorry, I misread it! Same with the other one.